src/Sulu/Bundle/AdminBundle/Resources/public/js/components/app/main.js

```diff
@@ -145,6 +147,72 @@ define(function() {
         },
 
         /**
+         * Initializes the handler for unsaved changes
+         */
+        initializeUnsavedChangesHandler: function() {
+            this.unsavedChanges = false;
+            this.bindCustomEventsForUnsavedChangesHandler();
+        },
+
+        /**
+         * Listen for changes regarding the save button
+         */
+        bindCustomEventsForUnsavedChangesHandler: function() {
+            this.sandbox.on('husky.toolbar.header.item.enable', function(button) {
```

Comment:
we probably should have introduced one at the start but I didn't want to change every bundle!